### PR TITLE
Raku support (try #2)

### PR DESCRIPTION
**Pull request by habere-et-dispertire**
I managed to exclude the markdown documentation in this pull request.
See: https://github.com/exercism/snippet-extractor/pull/53

### CODE REVIEW

**Comment by habere-et-dispertire** [comment]
I have no idea why the "Tests / test" is failing. The "Details" list no output.

**Comment by ErikSchierboom** [comment]
The build is currently failing due to rubocop linting errors. I've submitted a PR to your fork to fix it: https://github.com/habere-et-dispertire/snippet-extractor/pull/1

**Comment by habere-et-dispertire** [comment]
Thank you Erik. :)

Well, now I'm stumped. I thought I had it in this state and that the `rubocop` error was due to `Minitest::Test` which disliked the number of lines. The maximum appears to be 100, but `raku_test.rb` is 104 lines and somehow it is still passing.

Some Erik Magic I'm sure. :)

**Comment by ErikSchierboom** [comment]
Did you purposefully still have this as a draft PR?

**Comment by habere-et-dispertire** [comment]
Just being polite. :)

**Comment by ErikSchierboom** [comment]
Ah, okay. @m-dango could you look at this from a Raku viewpoint?

File: lib/languages/raku.txt [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+#`(+\j-->>)+\pj
+#`{+\j-->>}+\pj
+#`[+\j-->>]+\pj
+#`<+\j-->>>+\pj
+#`(\j-->>)\pj
+#`{\j-->>}\pj
+#`[\j-->>]\pj
+#`<\j-->>>\pj
+#
+=begin-->>=end
```
Comment (m-dango) [inline lib/languages/raku.txt:11]: I would suggest that this be `=begin comment-->>=end comment` with an additional `=comment`, unless there is some additional Pod you wish to cover?

https://docs.raku.org/language/pod#Pod6_comments

**Review by m-dango — CHANGES_REQUESTED** [review]

File: lib/languages/raku.txt (hunk 1 above)
Comment (habere-et-dispertire) [reply lib/languages/raku.txt]: I've added these - thank you. :)

And another thank you for [fixing the documentation around ｢｣](https://github.com/Raku/doc/issues/4110). I've added support for that too.

**Review by m-dango — APPROVED** [review]

**Comment by ErikSchierboom** [comment]
I'm reprocessing the existing snippets. Let me know if it worked.

**Comment by habere-et-dispertire** [comment]
It looks way better now. I'm looking at the [raku Hello World Community Solutions](https://exercism.org/tracks/raku/exercises/hello-world/solutions). Erik, thank you!
:-)

I'm still tempted to get rid of the `unit module` bit which feels like administrative detritus. If there is consensus, I'll add that in.

Aside: Am I right in thinking that Community Solutions for tutorial exercises are no longer reachable by the breadcrumb toolbar (but reappears if you go via a published solution)?

**Comment by ErikSchierboom** [comment]
They shouldn't even be visible at all :)

**Pull request merged**